core/common/src/IModel.ts

```diff
@@ -256,6 +256,13 @@ export class EcefLocation implements EcefLocationProps {
     this._transform.freeze();
   }
 
+  /** Returns true if this EcefLocation is not located at the center of the Earth.
+   * @alpha are locations very close to the center considered valid? What are the specific criteria?
+   */
+  public get isValid(): boolean {
+    return !this.origin.isZero;
```

Comment:
I think this should compare the length of the origin to a value which is closer to the radius of the earth with some tolerance that makes sense... not sure what the tolerance should be exactly but I think the length of the origin should be at least 6300km

https://en.wikipedia.org/wiki/Earth_radius#:~:text=Approximating%20the%20figure%20of%20Earth,polar%20radius%2C%20denoted%20b).

Reply:
Thanks, that's why I left it `alpha` with comments for future consideration. Our connectors appear to be writing ecef locations with origin at exactly (0, 0, 0) causing users to be unable to view their iModels.